packages/spinners/lib/index.ts

```diff
@@ -0,0 +1,96 @@
+import Spinnies, { SpinnerOptions, StopAllStatus } from "spinnies";
+
+const spinnies = new Spinnies();
+
+/**
+ * Add a new spinner with the given name.
+ */
+export function add(
+  name: string,
+  options: SpinnerOptions | string
+): SpinnerOptions {
+  if (typeof options === "string") {
+    options = {
+      text: options
+    };
+  }
+  return spinnies.add(name, options);
+}
+
+/**
+ * Get spinner by name;
+ */
+export function pick(name: string): SpinnerOptions {
+  return spinnies.pick(name);
+}
+
+/**
+ * Remove spinner with name.
+ */
+export function remove(name: string): SpinnerOptions {
+  return spinnies.remove(name);
```

Comment:
Does this need to be guarded?

Reply:
Good question. It [throws](https://github.com/jcarpanelli/spinnies/blob/90e33ee6384eab6d4783a506301ab3dbaee24b00/index.js#L75-L81) if the spinner name isn't known. However I think per my remark above, this is probably desirable in some cases, as it will help us to catch spinner lifecycle bugs at dev time.